server/graphql/types/src/types/mod.rs

```diff
@@ -155,3 +155,11 @@ impl IdResponse {
         &self.0
     }
 }
+
+pub struct OkResponse;
+#[Object]
+impl OkResponse {
+    pub async fn ok(&self) -> bool {
+        true
```

Comment:
Lol I always want a response object like this, realise it doesn't exist, so end up returning the node... but there wasn't a relevant pref node to return from the upsert response, so just implemented this. We don't use the mutation responses in the frontend anyway, I think this is okay?

Reply:
Is that cos you have to return something from GraphQL?